### ISSUE TITLE
settings.py layout

### ISSUE THREAD

**Issue by zeograd** [arabek/demovibes#6]
Back in django 1.3, the settings.py with local override was the norm, and this is used by demovibes.
However a more modern approach uses a settings/ directory in which we have a base.py containing most of the settings, a local.py which load base.py and then override settings for local deployment (adding the debug toolbar, enabling debugging, ...), a prod.py which overrides settings for production (database settings potentially, ...).
Secret values for production (password, secret hashing key, ..) can be read from the environment to avoid being committed publicly.
I propose to make those changes via a pull request, but it requires some adjustment on the environment in which it's deployed and I think it might be better to discuss implications beforehand.

Basically, it would impact deployment in that the environment should define DJANGO_SETTINGS_MODULE to be 'settings.local', 'settings.ci', ..., defaulting to 'settings.prod'.
It might require the PYTHONPATH variable to contain demovibes django root (so that 'settings.*' can be imported).
That should deprecate settings_local.py but for the sake of smoothing the transition, it could remain as final overriding source.

The rationale for this change is that we keep all settings (besides secrets) version controlled so that bug reproduction/debugging on different machines is eased. We also simplify settings.py by reducing conditions (installing debugbar if debug is enabled, logging to smtp if in production, using fakesink smtp on local platforms, ...).
